src/ResultSet.php

```diff
@@ -48,4 +53,21 @@ public function getIterator(): ArrayIterator
     {
         return new ArrayIterator($this->results);
     }
+
+    public function getErrors(): array
```

Comment:
Can you describe in phpdoc `@return` tag explicitly specify what items user can expect to be presented in return result?

Reply:
done

Reply:
As I can see in test (see below), the type should be an `string[][]` (array of arrays of strings), or for psalm `array<string, list<string>>`, not an `string[]`
```php
$this->assertSame(
            ['attribute1' => ['error1', 'error2']],
            $resultSet->getErrors()
);
```